_html/scss/modules/_homepage-burger-menu.scss

```diff
@@ -0,0 +1,136 @@
+.header--menu,
+.burger-button {
+    display: inline-block;
+    vertical-align: middle;
+}
+
+.link-menu {
+    right: 6rem;
+}
+
+.burger-button {
+    width: 25px;
+    height: 25px;
+    margin-left: 20px;
+    cursor: pointer;
+    z-index: 10;
```

Comment:
The z-index will be ignored here since the block has by default static positioning.